docs/system_overview/configuration.md

```diff
@@ -82,132 +128,126 @@ This file contains the following top level entries:
 
   - `pam_password_min_time` (optional) - Minimum allowed PAM password lifetime, in seconds
```

Comment:
Should PAM be listed here?

Seems like the section describing how to setup PAM would handle these. Same for Kerberos.

Reply:
removed